README.md

```diff
@@ -12,9 +12,44 @@ If bundler is not being used to manage dependencies, install the gem by executin
 
     $ gem install jupiter-ag-swap
 
+Set the SWAP_API_URL environment variable or the [default will be used](https://www.jupiterapi.com/):
+
+    $ export SWAP_API_URL=https://public.jupiterapi.com
```

Comment:
This doesn't appear to do anything. It looks like you've changed the default to QuickNode's Jupiter API wrapper with a 0.2% added fee and given no way to opt out.

I don't mind adding it as an option, but I'm trying really hard to assume you have positive intentions here.